docs/graph/authentication/uwp/WebAccountProviderConfig.md

```diff
@@ -0,0 +1,30 @@
+---
```

Comment:
Basically any basic enum/simple class as a helper to another, we should just have as part of the other doc for now, since we don't have any example or remarks to add on top of these (and eventually they'll just be auto-generated).